src/lib.rs

```diff
@@ -43,6 +43,23 @@ impl MongodbSessionStore {
     /// ```
     pub async fn connect(uri: &str, db: &str, coll_name: &str) -> mongodb::error::Result<Self> {
         let client = Client::with_uri_str(uri).await?;
+        let create_index = doc! {
```

Comment:
This should probably be an external initialize as suggested in the PR commit message. It would fit with the builder approach in the tide example https://github.com/jbr/tide-example/blob/main/src/main.rs#L24  and allow us to support both global and local options